examples/annotated-skaffold.yaml

```diff
@@ -195,6 +195,12 @@ deploy:
     #   delete: [""]
 
  # helm:
+    # helm can be passed additional option flags either 
+    # on every command (Global), on install (Install) or on update (Update).
```

Comment:
`udpate` should say `upgrade`

Reply:
I updated all the update->upgrade mistakes!